projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-view.model.ts

```diff
@@ -0,0 +1,19 @@
+import { ModelTemplatePropertyType } from '@hypertrace/dashboards';
+import { Model, ModelJson, ModelProperty, STRING_PROPERTY } from '@hypertrace/hyperdash';
+
+@Model({
+  type: 'table-widget-view'
+})
+export class TableWidgetViewModel {
+  @ModelProperty({
+    key: 'view',
+    type: STRING_PROPERTY.type
+  })
+  public view!: string;
```

Comment:
Is this the label that appears in the dropdown? If so, it's name could be updated to clarify.